src/services/requestsImport.js

```diff
@@ -1,123 +1,13 @@
 import { logger } from '@dojot/dojot-module-logger';
-import requestsDevice from './requestsDevice';
-import requestsTemplate from './requestsTemplate';
 import requestFlow from './requestsFlow';
 import Requests from '../utils/requests';
 import config from '../config';
 
 async function deleteAllData(token) {
```

Comment:
I think the name can be deleteAllFlows... What do you think?

Reply:
Hi Elias, how are you? From now on we are importing the devices and templates into the same json. We only sent one request to the device-manager. We are also considering ids on import.

Reply:
You're right, deleteAllFlows is more cohesive, I'll make the change